tracee-ebpf/tracee/tracee.go

```diff
@@ -889,12 +863,19 @@ func (t *Tracee) writeProfilerStats(wr io.Writer) error {
 	return nil
 }
 
+func (t *Tracee) Consume() (<-chan external.Event, error) {
```

Comment:
The model proposed implies that the ownership of the channel should be on the producer (indeed who writes on the channel should be responsible to close it). IMO this is a better design compared to a "pass channel from configuration" design, where the consumer is supposed to be the ownership of the channel. Moreover, in the model proposed by the commit we will still be able in future to add multiple channels (just return a struct/map of channels).